### ISSUE TITLE
Feature: Resolve `$__to` and `$__from` macros to `datetime` literals in queries

### ISSUE THREAD

**Issue by brycefisher** [grafana/azure-data-explorer-datasource#18]
The current macros for the start / end times are `long` values that represent unix time (in milliseconds I believe). Here's an example value: `1565389248181`.

However, in order to use those times in a valid ADX query, I need to convert it to a `datetime`. ADX datetime datatypes are 100-nanosecond units since the year 0. As a result, the first 6 lines of every ADX query I write looks like this:

```
let fromUnixTime = (t:long)
{ 
    (t / 1000) * 1sec + datetime(1970-01-01)
};
let start_time = fromUnixTime($__from);
let end_time = fromUnixTime($__to);
```

It would be really nice if `$__to` and `$__from` resolved to `datetime` literals instead of `long`s.

**Comment by erikbos** [comment]
I noticed https://github.com/grafana/azure-data-explorer-datasource/pull/15, looks like it might address this issue by adding support for:

```
`$__timeFrom` - Expands to `datetime(2018-06-05T18:09:58.907Z)`, the start time of the query.
`$__timeTo` - expands to `datetime(2018-06-05T20:09:58.907Z)`, the end time of the query.

```

**Comment by brycefisher** [comment]
Yes! That's amazing -- exactly the functionality I was looking for (at least according to the description in that PR)

Should I close this issue out or keeping to track progress on #15?
